client/src/api/resources/favoriteProjects.js

```diff
@@ -1,5 +1,7 @@
 export default {
-  index(filter = {}, page = {}) {
+  index(filterParams = {}, page = {}) {
+    const filter = filterParams;
+    filter.draft = false;
```

Comment:
This seems weird, too... seems like if a project is draft, it shouldn't show up. The filter state should depend on whether the user is an admin or not.